keyboards/tweetydabird/lotus58/config.h

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2021 TweetyDaBird
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+#include "config_common.h"
+
+// Not working correct in json implementation
+#define SPLIT_USB_DETECT
+#define SPLIT_USB_TIMEOUT 2000
+#define SPLIT_USB_TIMEOUT_POLL 10
```

Comment:
```suggestion

```

This is duplicated in info.json

Reply:
As the comment above specified, for whatever reason, the definition in .json makes the keyboard randomly time out on insertion/start-up with the exact same values that doesn't do so in config, so I was intending to make an 'issue' about that that, but had intended that after this PR (part of my impatience i guess). 

I'm opting to remove the .json part instead.

Reply:
That's fine for now.